js/util/stableSort.js

```diff
@@ -103,7 +103,7 @@ function descending(typeOfData) {
  * @param {function} getValue
  * @param {number} [direction=1]
  */
-function sort(index, getValue, direction) {
+function sort(index, getValue, direction, type) {
```

Comment:
We still need the ability to specify `comparitor.asc` and `comparitor.desc` per schema spec outlined in [#342](https://github.com/openfin/fin-hypergrid/pull/342).